var/spack/repos/builtin/packages/tau/package.py

```diff
@@ -106,6 +106,8 @@ class Tau(Package):
     )
     variant("dyninst", default=False, description="Activates dyninst support")
 
+    variant("disable-no-pie", default=False, description="Do not add -no-pie while linking.")
```

Comment:
Curious.  Is out worth noting in the description that the option relates to ubuntu?

Reply:
Added that this is for Ubuntu, thanks.